sdk/servicebus/Microsoft.Azure.ServiceBus/src/Primitives/ManagedIdentityTokenProvider.cs

```diff
@@ -12,7 +12,17 @@ namespace Microsoft.Azure.ServiceBus.Primitives
     /// </summary>
     public class ManagedIdentityTokenProvider : TokenProvider
     {
-        static AzureServiceTokenProvider azureServiceTokenProvider = new AzureServiceTokenProvider();
+        private readonly AzureServiceTokenProvider azureServiceTokenProvider;
```

Comment:
While somewhat trivial, I'd appreciate seeing a unit test or two validating the functionality.

Reply:
There's a documentation on how to run the tests (general: https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#to-test-1 and SB specific: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/servicebus/Microsoft.Azure.ServiceBus/README.md#running-tests) but nothing that would elaborate on the specifics. Is that guidance somewhere else, @jsquire?

Personally, I couldn't figure out how to test MSI locally with SB. I would appreciate some pointers/samples.

Reply:
>> Personally, I couldn't figure out how to test MSI locally with SB. I would appreciate some pointers/samples.

Unfortunately, not much that I can offer there; the MSI experience is difficult.   I was thinking more of simple unit tests to validate that your custom token provider was actually being used when you pass it in and that the default was used otherwise.   Trivial and not worth a large time investment, but we've now got branched execution that it would be nice to ensure in the face of potential refactoring in the future.

Reply:
>> Personally, I couldn't figure out how to test MSI locally with SB. I would appreciate some pointers/samples.

Unfortunately, not much that I can offer there; the MSI experience is difficult.   I was thinking more of simple unit tests to validate that your custom token provider was actually being used when you pass it in and that the default was used otherwise.   Trivial and not worth a large time investment, but we've now got branched execution that it would be nice to ensure in the face of potential refactoring in the future.